yb-voyager/src/utils/commonVariables.go

```diff
@@ -101,10 +101,11 @@ type DBObject struct {
 
 // TODO: support MinimumVersionsFixedIn in xml
 type Issue struct {
-	IssueType              string                          `json:"IssueType"`
+	IssueType              string                          `json:"IssueType"` //category: unsupported_features, unsupported_plpgsql_objects, etc
 	ObjectType             string                          `json:"ObjectType"`
 	ObjectName             string                          `json:"ObjectName"`
 	Reason                 string                          `json:"Reason"`
+	Type                   string                          `json:"-" xml:"-"` // identifier for issue type ADVISORY_LOCKS, SYSTEM_COLUMNS, etc
```

Comment:
ybd must be using this Issue struct, i guess when they read ConversionIssues.
Have we looked into the back compatiability aspect here(if required)?

Reply:
For now, I've excluded it from the json/xml. So no change to the structures. Just wanted to use it for filtering in code. Don't think we need to worry about backward compatibility at least for now?